src/NCronJob/Configuration/Builder/JobOption.cs

```diff
@@ -28,7 +28,7 @@ internal sealed class JobOption
     /// <summary>
     /// Startup Jobs will be executed once during the application startup before any other jobs.
     /// </summary>
-    public bool IsStartupJob { get; set; }
+    public bool? ShouldCrashOnStartupFailure { get; set; }
```

Comment:
Can it be "just" a `bool`? Given that there are only two states?

Reply:
I believe there are three states:
- Standard job => null
- StartupJob that shouldn't crash the startup => false
- StartupJob that should crash the startup => true

Reply:
Fair